### PR TITLE
[ADF-551] allow setting sorting in html

### PR DESCRIPTION
**Pull request by DenisVuyka**
- allow setting sorting in html
- disable permissions feature as causing problems

### CODE REVIEW

File: ng2-components/ng2-alfresco-datatable/src/components/datatable/datatable.component.ts [hunk 1]
```diff
@@ -236,7 +236,8 @@ export class DataTableComponent implements AfterContentInit, OnChanges {
         let event = new DataCellEvent(row, col, []);
         this.showRowActionsMenu.emit(event);
 
-        return this.checkPermissions(row, event.value.actions);
+        return event.value.actions;
+        // return this.checkPermissions(row, event.value.actions);
```
Comment (eromano) [inline ng2-components/ng2-alfresco-datatable/src/components/datatable/datatable.component.ts]: can we remove this commented code?

File: ng2-components/ng2-alfresco-documentlist/README.md [hunk 2]
```diff
@@ -192,6 +192,7 @@ The properties currentFolderId, folderNode and node are the entry initialization
 | `rowFilter` | `RowFilter` | | Custom row filter, [see more](#custom-row-filter). |
 | `imageResolver` | `ImageResolver` | | Custom image resolver, [see more](#custom-image-resolver). |
 | `allowDropFiles` | boolean | false | Toggle file drop support for rows (see **ng2-alfresco-core/UploadDirective** for more details) |
+| `sorting` | string[] | | Explicitly defines default sorting, i.e. ['name', 'desc'] |
```
Comment (eromano) [inline ng2-components/ng2-alfresco-documentlist/README.md]: can we add also an example with asc in this way users can understand the string used to change the order

File: ng2-components/ng2-alfresco-datatable/src/components/datatable/datatable.component.ts (hunk 1 above)
Comment (DenisVuyka) [inline ng2-components/ng2-alfresco-datatable/src/components/datatable/datatable.component.ts]: this code disables permissions functionality until we find out how to fix it. The intention is to uncomment it back in a separate PR when we have solution

File: ng2-components/ng2-alfresco-documentlist/README.md (hunk 2 above)
Comment (DenisVuyka) [inline ng2-components/ng2-alfresco-documentlist/README.md]: Ok, I'll update readme with more details

Comment (DenisVuyka) [inline ng2-components/ng2-alfresco-documentlist/README.md]: Updated readme with more details

File: ng2-components/ng2-alfresco-datatable/src/components/datatable/datatable.component.ts (hunk 1 above)
Comment (eromano) [inline ng2-components/ng2-alfresco-datatable/src/components/datatable/datatable.component.ts]: So at this point, I have two questions:
- this thing removes a functionality and there are no failing test? so are missing...
- this PR introduce another bug in order to solve another? Maybe is better fix the original problem instead to comment out the functionality?

Comment (DenisVuyka) [inline ng2-components/ng2-alfresco-datatable/src/components/datatable/datatable.component.ts]: I can't test the original problem as it is related to angular engine and change tracking crashes caused by the code I've commented out

Comment (DenisVuyka) [inline ng2-components/ng2-alfresco-datatable/src/components/datatable/datatable.component.ts]: restored the code

**Pull request merged**